models/rollup_economics/ethereum/rollup_economics_ethereum_schema.yml

```diff
@@ -0,0 +1,28 @@
+version: 2
+
+models:
+  - name: rollup_economics_ethereum_l1_fees
+    meta:
+      blockchain: ethereum
+      sector: rollup_economics
+      project: rollup_economics
+      contributors: niftytable
+    config:
+      tags: ['rollup', 'l2']
+    description: Fees paid by rollups for posting transaction data on Ethereum
+    columns:
+      - &name
+        name: name
+        description: "name of the rollup"
+      - &hash
```

Comment:
can you add a unique test on this column?

Reply:
like this?

    tests:
      - dbt_utils.unique_combination_of_columns:
          combination_of_columns:
            - name
            - hash